acos_client/tests/unit/v30/test_vrrpa.py

```diff
@@ -41,6 +41,12 @@ def expected_payload(self, vrid_val, threshold=1, disable=0):
                 }
             }
         }
+        if floating_ip:
```

Comment:
Just add this into the expected payload

Reply:
The existing test fails without this check. 
I need to have this check, because in test, the `expected_payload` method is setting default values of `threshold=1, disable=0`
whereas our main method does not set the default values.